src/current/v23.1/timestamp.md

```diff
@@ -208,27 +231,85 @@ ERROR: unimplemented: type conversion from TIMESTAMPTZ(5) to TIMESTAMPTZ(3) requ
 SQLSTATE: 0A000
 ~~~
 
-## Supported casting and conversion
+### Convert a timestamp to seconds since epoch
 
-`TIMESTAMP` values can be [cast]({% link {{ page.version.version }}/data-types.md %}#data-type-conversions-and-casts) to any of the following data types:
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT now()::int;
+~~~
 
-Type | Details
------|--------
-`DECIMAL` | Converts to number of seconds since the Unix epoch (Jan. 1, 1970).
-`FLOAT` | Converts to number of seconds since the Unix epoch (Jan. 1, 1970).
-`TIME` | Converts to the time portion (HH:MM:SS) of the timestamp.
-`INT` | Converts to number of seconds since the Unix epoch (Jan. 1, 1970).
-`DATE` | --
-`STRING` |  Converts to the date and time portion (YYYY-MM-DD HH:MM:SS) of the timestamp and omits the time zone offset.
+### Convert a timestamp to milliseconds since epoch 
 
-### Infinity `TIMESTAMP` casts
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT (now()::float*1000)::int;
+~~~
 
-CockroachDB currently does not support an `infinity`/`-infinity` representation for `TIMESTAMP` casts. Instead, `infinity::TIMESTAMP` evaluates to `294276-12-31 23:59:59.999999+00:00`, the maximum `TIMESTAMP` value supported, and `-infinity::TIMESTAMP` evaluates to `-4713-11-24 00:00:00+00:00`, the minimum `TIMESTAMP` value supported.
+### Convert a timestamp to microseconds since epoch 
 
-Note that this behavior differs from PostgreSQL, for which `infinity` is higher than any allowable `TIMESTAMP` value (including `294276-12-31 23:59:59.999999+00:00`), and `-infinity` is lower than any allowable `TIMESTAMP` value (including `-4713-11-24 00:00:00+00:00`).
+{% include_cached copy-clipboard.html %}
+~~~ sql
+SELECT (now()::float*1000000)::int;
+~~~
 
-For more details, see [tracking issue](https://github.com/cockroachdb/cockroach/issues/41564).
+### Convert seconds since epoch to timestamp if it's an int
```

Comment:
```suggestion
### Convert an `INT` seconds since epoch to `TIMESTAMP`
```

Reply:
Good catch, updated the language slightly differently as follows

> Convert an `INT` (seconds since epoch) to timestamp